app/javascript/mastodon/components/intersection_observer_article.js

```diff
@@ -21,6 +21,14 @@ export default class IntersectionObserverArticle extends ImmutablePureComponent
     isHidden: false, // set to true in requestIdleCallback to trigger un-render
   }
 
+  updateOnProps = [
+    'id',
+    'index',
+    'listLength',
+    'saveHeightKey',
+    'cachedHeight',
+  ]
+
```

Comment:
This change should be safe, because these are the props that are no longer compared:

- `intersectionObserverWrapper` - no sense in diffing this; it doesn't change anyway
- `onHeightChange` - listener, no sense diffing it
- `children` - this is already covered by `listLength`. There's no case I can think of where a toot would change but the `listLength` wouldn't change. Toots are added or subtracted from the list, but not replaced.